server/core/runtime/post_workflow_hook_runner.go

```diff
@@ -70,7 +70,8 @@ func (wh DefaultPostWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContex
 		}
 	}
 
-	wh.OutputHandler.SendWorkflowHook(ctx, fmt.Sprintf("%s\n", string(out)), true)
+	wh.OutputHandler.SendWorkflowHook(ctx, string(out), false)
```

Comment:
Thanks for the contribution and fix @Fabianoshz. Could you please add the test section to the pr body to describe how this was tested e2e for post and pre workflpws?

And if possible, could you also add a unit test for this to ensure this doesn't break pre and post workflow runs going forward?

Reply:
cc: @Fabianoshz friendly ping